webapp/src/pages/createInvitedGuest/components/createInvitedGuestComponent.js

```diff
@@ -139,6 +113,8 @@ class creatreInvitedGuestComponent extends Component {
     )
       return;
 
+    //TODO review this workflow, it seems dodgy. 
+    //createInvitedGuest always returns 400 (no role). Then addInvitedGuest creates the invitedguest.
```

Comment:
Really? it was working fine before... maybe a bug introduced by recent changes?

Reply:
Yeah I didn't see any new changes. I will test this properly on test, if it works I will remove the comment.